Serverless-Functions/homework/responses/1.5-blog.md

```diff
@@ -0,0 +1,60 @@
+<br><br/>
+
+Visual Studio Code (or VS code) was downloaded from this link: https://code.visualstudio.com
+
+Inside VS code, you downloaded four extensions: Azure Account, Azure App Service, Azure Functions, and Live Server. All of the Azure extensions allow you to work on your Azure Function App in VS code instead of working directly through the Microsoft portal. Live Server is a quick and temporary testing server, and you can use it to test HTML pages.
+<br><br/>
+
+For the GitHub portion of the week, there are two core things to remember: **repositories and committing changes.**
+
+Repositories (or repos) are essentially folders where you can store files of code. The repo of our learning lab was duplicated into your profile when you enrolled in the course so that you can commit changes and complete each lesson.
+
+For our learning lab, each lesson is placed inside an issue. Only when you complete the lesson (committing the necessary code), will the issue close and you can move on to the next issue. Don’t worry – committing changes is easier than it sounds.
+
+Here are the steps to cloning and committing changes to the learning lab repo (through GitHub desktop):
+
+1. Download GitHub desktop. It is here you commit changes to the learning lab to complete each lesson.
+2. To have access to the learning lab on GitHub desktop, the name of the learning lab repo needs to be entered in the top-left corner in the box with placeholder “Filter” and “Clone Repository” should be selected.
+3. Click “Pull Origin” once the repo is cloned to pull to desktop the most current version of the learning lab.
+4. Open VS code by clicking the button “Open in Visual Studio Code”. The files in the learning lab will be opened in VS code, and now you can complete the lesson (adding the necessary code). When you are done making changes in VS code, save the file and return to GitHub desktop.
+5. The change will be shown. Add a summary and description of your change and then click “Commit to main”. Next, click “Push origin”, to push the change made in VS code to the website.
+
+*Remember, each step and lesson is posted on the repo on github.com. You will commit changes when the lesson instructs you to, and once you click “Push origin” and refresh the GitHub page, the issue will close and you can move on.*
+
+You don’t have to commit changes on GitHub desktop, though.
+
+Here are the steps to committing code on the GitHub website:
+
+1. In the learning lab repo, there should be a button labeled “Add file” – click it.
+2. Click the dropdown option “Create new file”.
+3. You can add your code there, and when you are finished, you can click “Commit new file”
+4. If you click the second dropdown option under the “Add file” button (“Upload files”), you can upload a file from VS code that is saved in your computer.
+<br><br/>
+
+Deploying an Azure Function was the biggest task of the week.
+
+Here are the steps:
+
```

Comment:
Once again, great steps, but try adding some pictures to help! Azure also has a bunch of instructions on how to create function apps, so also link them here